### PR TITLE
MTA-2162-Remove-Quick-Fix

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTA-2162](https://issues.redhat.com/browse/MTA-2162)

### PREVIEW

* [4.1. Reviewing issues](https://deploy-preview-832--windup-documentation.netlify.app/docs/vs-code-extension-guide/master/#vs-code-extension-reviewing-issues_vsc-extension-guide)
* [4.2. Resolving issues](https://deploy-preview-832--windup-documentation.netlify.app/docs/vs-code-extension-guide/master/#vs-code-extension-resolving-issues_vsc-extension-guide)

### CODE REVIEW

File: docs/topics/vs-code-extension-resolving-issues.adoc [hunk 1]
```diff
@@ -6,14 +6,12 @@
 [id="vs-code-extension-resolving-issues_{context}"]
 = Resolving issues
 
-You can resolve issues by doing one of the following:
-
-* Using a _Quick Fix_ automatic code replacement
-* Editing the code of a file with a hint
+You can resolve issues by editing the code of a file with a hint
 
 You can keep track of resolved issues by using the *Delete* or *Mark as Complete* options. Files marked deleted or completed will be analyzed again the next time you analyze a project that contains them.
```
Comment (midays) [inline docs/topics/vs-code-extension-resolving-issues.adoc:11]: `You can keep track of resolved issues by using the *Delete* or *Mark as Complete* options. Files marked deleted or completed will be analyzed again the next time you analyze a project that contains them.`
This needs to be removed, or at least commented out, this functionality is not available in vscode.

File: docs/topics/vs-code-extension-reviewing-issues.adoc [hunk 2]
```diff
@@ -18,4 +18,4 @@ You can use the {ProductShortName} extension icons to prioritize issues based on
 
 ** image:vs_mandatory.png[Mandatory] : The issue must be fixed for a successful migration.
 ** image:vs_potential.png[Warning] : The issue might need to be addressed during migration.
-** image:vs_optional.png[Optional or Quick Fix] : The issue is optional to fix for migration. This icon is also used to indicate any Quick Fixes available for an issue.
+// ** image:vs_optional.png[Optional or Quick Fix] : The issue is optional to fix for migration. This icon is also used to indicate any Quick Fixes available for an issue.
```
Comment (midays) [inline docs/topics/vs-code-extension-reviewing-issues.adoc:21]: This is still present in the preview.

**Review by midays — CHANGES_REQUESTED** [review]
@anarnold97

File: docs/topics/vs-code-extension-resolving-issues.adoc (hunk 1 above)
Comment (anarnold97) [inline docs/topics/vs-code-extension-resolving-issues.adoc:11]:
```suggestion
```

File: docs/topics/vs-code-extension-reviewing-issues.adoc (hunk 2 above)
Comment (anarnold97) [reply docs/topics/vs-code-extension-reviewing-issues.adoc:21]: image

Comment (anarnold97) [inline docs/topics/vs-code-extension-reviewing-issues.adoc:21]:
```suggestion
```

**Review by midays — APPROVED** [review]
LGTM, 
Thank you!

**Pull request merged**